Formula/cntlm.rb

```diff
@@ -58,4 +58,30 @@ def plist
       </plist>
     EOS
   end
+
+  test do
+    assert_match "version #{version}", shell_output("#{bin}/cntlm -h 2>&1", 1)
+
+    bind_port = free_port
+    (testpath/"cntlm.conf").write <<~EOS
+      # Cntlm Authentication Proxy Configuration
+      Username	testuser
+      Domain		corp-uk
+      Password	password
+      Proxy		localhost:#{free_port}
+      NoProxy		localhost, 127.0.0.*, 10.*, 192.168.*
+      Listen		#{bind_port}
+    EOS
+
+    begin
+      pid = fork do
+        exec "#{bin}/cntlm -c #{testpath}/cntlm.conf -v"
+      end
+      sleep 2
+      assert_match /502 Parent proxy unreacheable/, shell_output("curl -s localhost:#{bind_port}")
+    ensure
+      Process.kill("TERM", pid)
+      Process.wait(pid)
+    end
```

Comment:
This is handled by brew, no need for it.

Reply:
Thanks, @SMillerDev. I had seen it as a common design pattern in other formulas. Removed the exception handling and force pushed the update.